### DISCUSSION TITLE
How to Style the selected and hovered items in the Nav control

### DISCUSSION THREAD

**Discussion by mhomol** [microsoft/fluentui#23554] [Q&A] [answered]
I need help in styling the Nav component's hovered and selected states.

I have a `backgroundColor` supplied for `root` that is working great along with `color` being set in `linkText` and 'chevronIcon`, but I'd like to invert the color of the link on the hover and selected states so that it's the darker color I'm using for the background of the Nav.  This seems like it should be really simple but I'm completely stumped.  Can someone help me here?

Notice in the screenshot that the top group item is selected and the bottom item is being hovered over:
<img width="410" alt="Screen Shot 2022-06-14 at 9 08 14 PM" src="https://user-images.githubusercontent.com/680566/173717860-49adc037-6bf7-4605-9151-eb6844d1deb2.png">

**Comment by khmakoto** [comment, marked as answer]
Hi @mhomol, this is unfortunately a quirk in how `Nav` is styled, but I think you should be able to achieve this similar to how I did in [this codepen](https://codepen.io/khmakoto/pen/gOvEOxE?editors=0010).

Let me know if you need more help.

**Comment by mhomol** [comment]
OMG thank you so very much!  This is precisely what I was looking for

**Reply by khmakoto** [reply]
Happy to help!

**Comment by D-Impulse** [comment]
Thank you very much @khmakoto, you are Saviour 😍🙌

**Reply by khmakoto** [reply]
Glad this answer is still helpful!
